src/main/drivers/accgyro.h

```diff
@@ -25,9 +25,8 @@ typedef struct gyro_s {
     sensorGyroInitFuncPtr init;                             // initialize function
     sensorReadFuncPtr read;                                 // read 3 axis data function
     sensorReadFuncPtr temperature;                          // read temperature if available
-    sensorInterruptFuncPtr intStatus;
     float scale;                                            // scalefactor
-    uint32_t targetLooptime;
+    uint32_t gyroSamplingRate;
```

Comment:
`gyroSamplingRate` should be `gyroSamplingInvterval` - a rate is measured in Hz, whereas an interval is a time period. A value of 125, for example, does not mean a sampling rate of 125Hz, it means a sampling interval of 125us.

This code can get confusing enough without the variable names misdirecting the thought processes.